bessctl/conf/perftest/chain.bess

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2014-2016, The Regents of the University of California.
+# Copyright (c) 2016-2017, Nefeli Networks, Inc.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+# list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+# this list of conditions and the following disclaimer in the documentation
+# and/or other materials provided with the distribution.
+#
+# * Neither the names of the copyright holders nor the names of their
+# contributors may be used to endorse or promote products derived from this
+# software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+import sys
+import time
+
+psize = int($BESS_PKT_SIZE!'60')
+print('Using packet size %d (envvar "BESS_PKT_SIZE")' % psize)
+
+def create_pipeline(chain_len):
+    last = Source(pkt_size=psize)
+
+    for i in range(chain_len):
+        now = Bypass()
+        last -> now
+        last = now
+
+    last -> Sink()
+
+def measure(duration):
+    tcs = bess.list_tcs().classes_status
+    root_tc = None
+    for tc in tcs:
+        if not getattr(tc, 'parent'):
+            root_tc = getattr(tc, 'class')
+    if not root_tc:
+        raise Exception('Fail to find root tc')
```

Comment:
Should we turn this little chunk of code into a method available in unit tests and an exported function in old style tests?  There's one copy in `bessctl/test_utils.py` and now this here and below...

Or, perhaps `measure` should be the exported function.

Reply:
Right, those are quite redundant.